FoodSuggestions/app/src/main/res/layout/fragment_ingredients.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.IngredientsFragment">
+
+        <androidx.recyclerview.widget.RecyclerView
```

Comment:
We can have directly RecyclerView as root.

Reply:
Fixed